### ISSUE TITLE
ESP8285 can't use ap_if.config "can't set AP config"

### ISSUE THREAD

**Issue by mindforger** [micropython/micropython#3813]
i did the following

>ap_if = network.WLAN(network.AP_IF)
ap.active(False)
ap.config(essid="someID", password="somePassword")

but i always get "can't set AP config", no matter what parameter i try to use

MicroPython v1.9.4-8-ga9a3caad0 on 2018-05-11; ESP module with ESP8266 

flashed on an plain ES8285 Module

listdir is also doing funky stuff on this module, getting a crap ton of " '\x00\x00\x00\x00\x00\x00\x00\x00.\x00\x00\x00'," as answer + the folders and file that icreated for a test

**Comment by mindforger** [comment]
how do i do that? i did not find a methode "format" yet in the docs

**Comment by dpgeorge** [comment]
> stop, i CAN change the essid but i can not change the password

Did you try erasing completely the flash (esptool.py erase_flash, or "make erase") before putting MicroPython on it?

**Comment by mindforger** [comment]
Yes i erased it, but on an 8285 it seems to break the default filesystem, i formated it and created a new empty Boot.py

**Comment by fjand5** [comment]
you have to active befor config.

> 
> import network
> ap_if = network.WLAN(network.AP_IF)
> ap_if.active(False)
> ap_if.config(essid="someID", password="somePassword")
>

**Comment by Medve01** [comment]
I have the same issue. It's a brand new board, flash erased before uploading firmware.
Board starts with a default AP set to "micropython-XXXX". I use the code from previous comment and I get the same error "OSError: can't set AP config".

I cannot change AP config...
